gunicorn.conf.py

```diff
@@ -0,0 +1,9 @@
+bind = "0.0.0.0:8000"
+threads = 4
+timeout = 400
+loglevel = "debug"
+protocol = "proxy"
+syslog = True
+forwarded_allow_ips = "10.0.2.100,127.0.0.1"
```

Comment:
`forwarded_allow_ips = "127.0.0.1"`

I would suggest having deployment-specific configurations in their respective repo.

Reply:
Sounds good, but in a specific future PR.

Reply:
But should not the configuration of gunicorn be handled in servers rather than be dictated by code in the repo? If we want to increase the number of workers we shouldn't have to make a PR to cg

Reply:
That would be a even bigger change. And I do not know if it would make sense to move it away from the actual app. Either way it is definitively out of scope.

If you agree - could you approve this PR?

Reply:
@ahdamin @Vince-janv Bump!